core/templates/pages/exploration-editor-page/services/exploration-states.service.ts

```diff
@@ -60,6 +62,7 @@ angular.module('oppia').factory('ExplorationStatesService', [
     var stateDeletedCallbacks = [];
     var stateRenamedCallbacks = [];
     var stateInteractionSavedCallbacks = [];
+    var _refreshGraphEventEmitter = new EventEmitter();
```

Comment:
Please don't add underscore, use a `/** @private */` pragma instead.

```suggestion
    /** @private */
    var refreshGraphEventEmitter = new EventEmitter();
```

Reply:
+cc @seanlip wasn't it decided that we will have underscore prefixes for private variables?

Reply:
Done

Reply:
Yup that's right. I don't remember the private pragma thing ever being discussed... it's not a convention AFAIK. The point of the underscore is so it's clear at the callsites when there is a violation.